### PR TITLE
Mock collectibles

### PR DESCRIPTION
**Pull request by marijnschilling**
When in debug I create a mockTeapot and fetch the collectible and collectibles on that one. 

I wonder if this debug check i made is safe and clear enough, or should it be something that should be able to enable?

Right now the images don't load, I'm not sure why not. Might be the same reason that the MessageRequest images aren't loading, so will take that on on the similar issue.

### CODE REVIEW

File: Toshi/APIClients/EthereumAPIClient.swift [hunk 1]
```diff
@@ -208,7 +208,14 @@ final class EthereumAPIClient {
     }
 
     func getCollectible(address: String = Cereal.shared.paymentAddress, contractAddress: String, completion: @escaping ((Collectible?, ToshiError?) -> Void)) {
-        self.activeTeapot.get("/v1/collectibles/\(address)/\(contractAddress)") { result in
+        var collectiblesTeapot = self.activeTeapot
+
+        // If we are on debug we will mock out the collectible data since they are only on production
```
Comment (designatednerd) [inline Toshi/APIClients/EthereumAPIClient.swift]: what if we're debugging and we actually WANT production? Would it make sense to check both that and the bundle ID is not the prod bundle?

File: Toshi/APIClients/EthereumAPIClient.swift [hunk 2]
```diff
@@ -208,7 +208,14 @@ final class EthereumAPIClient {
...
+        #if DEBUG
+        collectiblesTeapot = MockTeapot(bundle: Bundle(for: EthereumAPIClient.self), mockFilename: "getACollectible")
```
Comment (designatednerd) [inline Toshi/APIClients/EthereumAPIClient.swift]: can you please indent this one more tab for readability?

File: Toshi/APIClients/EthereumAPIClient.swift (hunk 1 above)
Comment (designatednerd) [reply Toshi/APIClients/EthereumAPIClient.swift]: [Nevermind, `#DEBUG` is an `other swift flags` var that's passed in so this comment is not relevant]

Comment (marijnschilling) [reply Toshi/APIClients/EthereumAPIClient.swift]: Should i maybe explain this better in the comment?

Comment (designatednerd) [reply Toshi/APIClients/EthereumAPIClient.swift]: Couldn't hurt

File: Toshi/APIClients/EthereumAPIClient.swift [hunk 3]
```diff
@@ -208,7 +208,14 @@ final class EthereumAPIClient {
     }
 
     func getCollectible(address: String = Cereal.shared.paymentAddress, contractAddress: String, completion: @escaping ((Collectible?, ToshiError?) -> Void)) {
-        self.activeTeapot.get("/v1/collectibles/\(address)/\(contractAddress)") { result in
+        var collectiblesTeapot = self.activeTeapot
+
+        // If we are on debug (specified in "other swift flags") we will mock out the collectiblesince they are only on production
```
Comment (designatednerd) [inline Toshi/APIClients/EthereumAPIClient.swift]: [optional] `collectiblesince` -> `collectiblesSince`

**Pull request merged**